ui/src/VoxettaApp.js

```diff
@@ -33,14 +36,52 @@ export class VoxettaApp extends LitElement {
 
     static get properties() {
         return {
-            state: {type: String}
+            state: {type: String},
+            promptState: {type: Boolean},
```

Comment:
I don't see this.promptState being used anywhere, only being re-assigned, but never read. Remove?

Reply:
Add documentation?

Reply:
added

Reply:
Instead of toggling promptState trigger the getNewPrompt, you can call it directly.
Keep a reference to the vox-prompt component with @query 

And in your skip button handler you can do this.promptComponent.getNewPrompt();

Ping if any of this is unclear.

Reply:
Oh I didn't know this was an option! I might be mistaken but from what I read online the @query is only for TypeScript, but I used querySelector in a function in vox-app to get rid of the promptState property--currently working on cleaning it up and will push an update soon